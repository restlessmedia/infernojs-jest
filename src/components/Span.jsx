import { Component } from 'inferno';

class Span extends Component {
  render() {
    return <span>{this.props.children}</span>
  }
}

export default Span;
