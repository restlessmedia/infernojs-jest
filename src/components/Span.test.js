import Span from './Span';
import { renderToSnapshot } from 'inferno-test-utils';

describe('Span', () => {

  it('should render', () => {
    const props = {};
    const renderedTree = renderToSnapshot(<Span {...props} />);

    expect(renderedTree).toMatchSnapshot();
  });
});
