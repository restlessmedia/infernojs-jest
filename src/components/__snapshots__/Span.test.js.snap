// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Span should render 1`] = `<span />`;
